docker/check_docker-compose.sh

```diff
@@ -21,6 +22,11 @@ if ! docker compose version > /dev/null 2>&1 ; then
 	exit 4
 fi
 
+# verify that docker compose & docker-compose aren't both installed
+if docker-compose version | grep '^docker-compose .* [0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*' ; then
```

Comment:
This check will print out to the cli. I believe we want to add a `-q` to the grep command.